### PR TITLE
Upgrade to Scala 2.12.11

### PR DESCRIPTION
**Pull request by olafurpg**

### CODE REVIEW

File: build.sc [hunk 1]
```diff
@@ -61,7 +62,7 @@ trait AmmInternalModule extends mill.scalalib.CrossSbtModule{
         Nil
 
     val extraDir2 = PathRef(
-      if (sv == "2.12.10" || sv == "2.13.1") millSourcePath / "src" / "main" / "scala-2.12.10-2.13.1+"
+      if (isScala2_12_10OrLater(sv)) millSourcePath / "src" / "main" / "scala-2.12.10-2.13.1+"
```
Comment (alexarchambault) [inline build.sc]: I think `2.13.0` has to be explicitly excluded here (the extra dir is for `2.12.{10, 11, …}`, and `2.13.{1, …}` excluding `2.13.0`).

Comment (olafurpg) [reply build.sc]: The variable name may be confusing, the set does not contain 2.13.0

Comment (alexarchambault) [reply build.sc]: Oh my bad 😬

**Comment by olafurpg** [comment]
Not sure if the following failure is related
```
X ammonite.session.AdvancedTests.loadURL 2033ms 

  utest.AssertionError: assert{

  error: String = error while loading ShrinkLowPriority, class file '/maven2/org/scalacheck/scalacheck_2.12/1.14.0/scalacheck_2.12-1.14.0.jar(org/scalacheck/ShrinkLowPriority.class)' is broken

  (class java.lang.ArrayIndexOutOfBoundsException/0)

  error while loading Gen, class file '/maven2/org/scalacheck/scalacheck_2.12/1.14.0/scalacheck_2.12-1.14.0.jar(org/scalacheck/Gen.class)' is broken

  (class java.lang.ArrayIndexOutOfBoundsException/0)
```

File: readme/Footer.scalatex [hunk 2]
```diff
@@ -123,6 +123,10 @@
 
 
   @sect{Changelog}
+    @sect{2.0.1}
```
Comment (alexarchambault) [inline readme/Footer.scalatex]: I think the next release should be `2.0.5`.
```suggestion
    @sect{2.0.5}
```

**Comment by olafurpg** [comment]
@alexarchambault Thank you for updating! Feel free to commandeer this PR

**Comment by alexarchambault** [comment]
Thanks @olafurpg! I merged this via https://github.com/lihaoyi/Ammonite/pull/1057.

**Pull request closed without merging**